oyatplum/react-todo/src/App.css

```
@@ -0,0 +1,69 @@
+.App {
+    margin:0;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: rgb(239, 226, 205);
+    height: 100vh;
+  }
+  .title{
+    height: 50px;
+    width: 340px;
+    border-radius: 15px;
+    padding: 3px;
+    display: flex;
+    align-items: center;
+  }
+  .Container {
+    padding: 20px;
+    width: 350px;
+      height: 650px;
+    box-shadow: rgb(0 0 0 / 20%) 0 0 20px;
+    margin: auto;
+    background-color: white;
+    border-radius: 20px;
+  }
+  .calendar-box{
+    height: 270px;
+    width: 340px;
+    background-color: rgb(246, 206, 206);
+    padding: 5px;
+    border-radius: 15px;
+    margin: 15px auto 15px auto;
```

```suggestion
      margin: 15px auto;
```
로 줄일 수 있습니다. `margin : (상하) (좌우)`